src/Entity/ConsularManagedArea.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class ConsularManagedArea
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @var ReferentTag[]|ArrayCollection
+     *
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\ReferentTag")
```

Comment:
1) should be `@ManyToOne`, no? 
2) `targetEntity="AppBundle\Entity\ConsularDistrict"`